lib/server.js

```diff
@@ -45,7 +45,11 @@ var start = function(injector, config, launcher, globalEmitter, preprocess, file
     if (e.code === 'EADDRINUSE') {
       log.warn('Port %d in use', config.port);
       config.port++;
-      webServer.listen(config.port);
+      if (config.listen) {
```

Comment:
I think now we can remove `if (config.lister) {` and simple write `webServer.lister(config.port, config.lister)` and `webServer.lister(config.port,config.lister, startCallback)`

Thanks!